docs/user_workloads/templates.md

```diff
@@ -160,12 +160,9 @@ items:
       metadata:
         name: rheltinyvm
       spec:
-        pvc:
+        storage:
```

Comment:
It's a mistake :sweat_smile: , the idea is to remove the accessMode from all of them

Reply:
Well some are not a mistake:
- I kept `accessMode: ReadWriteMany` because it is not the default option
- I removed the resource requests only when the source is another PVC (i.e. we're cloning it), because CDI can know the necessary resources based on the source PVC.

Reply:
Actually, the only `ReadWriteOnce` access mode that you left was in a DV using local storage class, so I think it was ok to have it in that case.